server/plugins/CMakeLists.txt

```diff
@@ -69,6 +69,47 @@ foreach(plugin ${plugin_sources})
 	list(APPEND plugins ${plugin_name})
 endforeach(plugin)
 
+if(AUDIOAPI STREQUAL bela)
+	# The Bela lib will have its own set of libraries and includes
+	# However, the Bela audio backend for SC has some direct calls to Xenomai
+	# Therefore we need to get flags for those as well
+	find_package(Xenomai)
+	if(NOT XENOMAI_FOUND)
+		message(FATAL_ERROR "Bela selected as audio API, but Xenomai development files not found")
+	endif()
+	find_package(Bela)
+	if(NOT BELA_FOUND)
+		message(FATAL_ERROR "Bela selected as audio API, but Bela development files not found")
+	endif()
+	message(STATUS "bela libs: ${BELA_LIBRARIES}")
+	if (CMAKE_COMPILER_IS_GNUCXX OR CMAKE_COMPILER_IS_CLANG)
+		# recommended compile flags for beaglebone etc; set here because bela api flag directly implies the architecture
+		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${BELA_C_FLAGS}")
+		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${BELA_CXX_FLAGS}")
+	endif()
+endif()
+
+if (BELA_FOUND)
+	add_library(BELAUGens MODULE
+	  BELAUGens.cpp
+# 	  ${BELA_SOURCES}
```

Comment:
I'd suggest to either describe why the commented out code is here, or remove it. The same comment applies a [few  line below](https://github.com/supercollider/supercollider/pull/5295/files#diff-3d1ccc9e1c72318a4384e2c1430e41a1b02019a0352cff7dbe4e6791eac2f9b7R103)

Reply:
thanks 2abebccf896c821382c6fa4e3040db09a9de82f1